pages/mesosphere/dcos/services/hive-metastore/1.1.0-3.0.0/release-notes/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes  
+excerpt: Discover the new features, updates, and known limitations in this release of the Hive Metastore Service
+title: Release Notes 
+menuWeight: 10
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+---
+# Release Notes for the {{ model.techName }} Service version 1.1.0-3.0.0
+
+## Updates
```

Comment:
Does this match the update instruction on the updates page? The updates page discusses several steps the user must take before updating; I fear that users might think that updating the service can be accomplished with only one command. I suggest that we replace the command shown with a link to the updates page, and that this command and any changes to the update process be added to that updates page.

Reply:
Yes it make sense to divert users to `Update` section for complete update instructions.
As discussed created another [PR](https://github.com/mesosphere/dcos-docs-site/pull/2753) for the same.